src/app/core/services/sage300/sage300-configuration/sage300-connector.service.ts

```
@@ -37,4 +37,9 @@ export class Sage300ConnectorService {
     return this.apiService.post(`/workspaces/${this.storageService.get('workspaceId')}/credentials/sage300/`, data);
   }
 
+  @Cacheable()
+  checkSage300TokenHealth(workspaceId: string): Observable<{}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)